course/src/isometry.cc

```diff
@@ -0,0 +1,82 @@
+#include "isometry.h"
+
+
+namespace cppcourse {
+
+const Vector3 Vector3::kUnitX = {1., 0., 0.};
+const Vector3 Vector3::kUnitY = {0., 1., 0.};
+const Vector3 Vector3::kUnitZ = {0., 0., 1.};
+const Vector3 Vector3::kZero = {0., 0., 0.};
+
+
+Vector3 Vector3::operator+(const Vector3 &other) const{
+    return Vector3(x_ + other.x_, y_ + other.y_, z_ + other.z_);
+};
+Vector3 Vector3::cross(const Vector3 & other) const{
+    Vector3 output = Vector3::kZero;
```

Comment:
No need to initialize it. It will use the default constructor of your class.

Reply:
Removed initialization